src/interfaces/interfaces.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
This folder and file seem to be used only in one test file. Can we maybe remove it and copy the relevant interfaces?

Reply:
this will be used by more test cases as i think we'll want these in each of the pool tests

Reply:
mmm not super-critical I guess, but if you think it will be re-shared then I think it's fine if it will be inside a `test` folder (along with the tests if we decide to re-arrange it like that).
Currently it's a bit confusing as most of the time when projects have an `interfaces` folder it's for defining new interfaces that the project introduces.